src/WebJobs.Extensions.DurableTask/ActorScheduler/ActorRef.cs

```diff
@@ -0,0 +1,77 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.Serialization;
+using System.Text;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.WebJobs
+{
+    /// <summary>
+    /// A reference to an actor, used to identify actor instances.
+    /// </summary>
+    public struct ActorRef : IEquatable<ActorRef>, IComparable
+    {
+        /// <summary>
+        /// Create an actor reference for an actor.
+        /// </summary>
+        /// <param name="actorClass">the name of the actor class.</param>
+        /// <param name="actorKey">the actor key.</param>
+        public ActorRef(string actorClass, string actorKey)
+        {
+            if (string.IsNullOrEmpty(actorClass))
+            {
+                throw new ArgumentException("invalid actor reference: actor class must not be a null or empty string", actorClass);
+            }
+
+            this.ActorClass = actorClass;
+            this.ActorKey = actorKey;
+        }
+
+        /// <summary>
+        /// The name of the actor class.
+        /// </summary>
+        [JsonProperty("class")]
+        public string ActorClass { get; set; }
+
+        /// <summary>
+        /// The actor key.
```

Comment:
I would add some additional description here. For example, we should call out that this is expected to be some _unique_ key for the actor.